portfolio.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>The Personal Portfolio of Me</title>
+
```

Comment:
inconsistent white space

Reply:
I separated out the link tags I have copied from elsewhere to insert into my code (the three below)